### ISSUE TITLE
Warnings accessibility on bottom tab

### ISSUE THREAD

**Issue by paavovaananen-cgi** [fmidev/weather-app#361]
Bottom tab warnings status/severity should be visible to screen reader. Sufficient solution would be to indicate if there are active warnings.
